packages/device_info_plus/device_info_plus/example/integration_test/device_info_plus_test.dart

```diff
@@ -65,4 +66,121 @@ void main() {
       }
     }
   });
+
+  testWidgets('iOS', (WidgetTester tester) async {
+    if (Platform.isIOS) {
+      expect(iosInfo.name, isNotNull);
+      expect(iosInfo.systemName, equals('iOS'));
+      expect(iosInfo.systemVersion, isNotNull);
+      expect(iosInfo.model, equals('iPhone'));
+      expect(iosInfo.localizedModel, equals('iPhone'));
+
+      expect(iosInfo.identifierForVendor, isNotNull);
+      expect(iosInfo.isPhysicalDevice, isNotNull);
+
+      expect(iosInfo.utsname.sysname, equals('Darwin'));
+      expect(iosInfo.utsname.nodename, isNotNull);
+      expect(iosInfo.utsname.release, isNotNull);
+      expect(iosInfo.utsname.version, isNotNull);
+      expect(iosInfo.utsname.machine, isNotNull);
+    }
+  });
```

Comment:
At the moment, there's no iOS integration testing setup, only Android is set up, so we cannot have this test (as it will do nothing)